### PR TITLE
enable filter service plans, service offerings and marketplace by a s…

### PR DESCRIPTION
**Pull request by sigalmaya**
Enable filtering service plans, service offerings, and marketplace by the supported platform.

### CODE REVIEW

File: internal/cmd/offering/marketplace_test.go [hunk 1]
```diff
@@ -143,6 +143,7 @@ var _ = Describe("Marketplace command test", func() {
 		})
 	})
 
+
```
Comment (eyalnathan) [inline internal/cmd/offering/marketplace_test.go:146]: shouldn't there be a test here? one that verifies that both service offerings and plans are loaded with the env parameter?

File: pkg/query/parameters.go [hunk 2]
```diff
@@ -12,6 +12,7 @@ type Parameters struct {
 	FieldQuery    []string
 	LabelQuery    []string
 	GeneralParams []string
+	Environment		string
```
Comment (eyalnathan) [inline pkg/query/parameters.go:15]: go fmt this file to fix the indentation here and missing spaces below

File: pkg/smclient/client.go [hunk 3]
```diff
@@ -252,15 +252,13 @@ func (client *serviceManagerClient) ListPlatforms(q *query.Parameters) (*types.P
 func (client *serviceManagerClient) ListOfferings(q *query.Parameters) (*types.ServiceOfferings, error) {
 	offerings := &types.ServiceOfferings{}
 	err := client.list(&offerings.ServiceOfferings, web.ServiceOfferingsURL, q)
-
```
Comment (eyalnathan) [inline pkg/smclient/client.go:255]: why?

Comment (sigalmaya) [reply pkg/smclient/client.go:255]: put the line back

**Review by eyalnathan — APPROVED** [review]

**Pull request merged**